dnf-plugins-core.spec

```diff
@@ -1,6 +1,6 @@
 %{?!dnf_lowest_compatible: %global dnf_lowest_compatible 1.1.4}
 %{?!dnf_not_compatible: %global dnf_not_compatible 2.0}
-%global hawkey_version 0.6.1
+%global hawkey_version 0.7.0
```

Comment:
I understand your doubts but that's only about a branch name. I don't see any difference between using master as a stable and cherry-pick bits to experimental or vice versa. This approach was suggested by @jsilhan, so he can provide us his POW.